src/os/unix/font_unix.cpp

```diff
@@ -39,6 +39,19 @@ static std::tuple<std::string, std::string> SplitFontFamilyAndStyle(std::string_
 	return { std::string(font_name.substr(0, separator)), std::string(font_name.substr(begin)) };
 }
 
+/**
+ * Get language string for FontConfig pattern matching.
+ * @param language_isocode Language's ISO code.
+ * @return Language code for FontConfig.
+ */
+static std::string GetFontConfigLanguage(const std::string &language_isocode)
+{
+	/* Fontconfig doesn't handle full language isocodes, only the part
```

Comment:
I'm not exactly sure what you mean by this.  The fontconfig library will parse "en_GB" without any issues.  I don't see any fonts when I use that as the language set because all of the fonts on my system use "en". 

Scrolling through the list shows me that fonts having a  "xx-xx" language code is very rare.

Reply:
```$ fc-list :lang=en | wc -l
1883
$ fc-list :lang=en_GB | wc -l
0```

Reply:
The comment is misleading. No issues with parsing that text.  Just none of the fonts use it.

Reply:
The comment does not mention anything about parsing.